### PR TITLE
Update style-guide.md

### PR DESCRIPTION
**Pull request by logankilpatrick**
Fixing grammatical error.

### CODE REVIEW

**Comment by ViralBShah** [comment]
Actually, most of this was fine before, and shouldn't be changed, in my opinion. The only exception being`,` after `cases` is a good idea.

**Comment by logankilpatrick** [comment]
The good folks at Grammarly made those suggestions to me. I oftentimes find that the most human-readable English is not perfect English.  Nonetheless, I wanted to make the suggestions! feel free to take whichever you believe makes sense.

**Comment by logankilpatrick** [comment]
Things like adding "the" in front of "idomatic" would techncially make it gramatically correct, but may not actually help anyone.

**Comment by ViralBShah** [comment]
Too much machine learning! ;-)

File: doc/src/manual/style-guide.md [hunk 1]
```diff
@@ -412,4 +412,4 @@ julia> h(1)
 ```
 
 Thus, use `Int` literals when possible, with `Rational{Int}` for literal non-integer numbers,
```
Comment (Keno) [inline doc/src/manual/style-guide.md]: I'd strike the whole dependent clause and if anything add `Thus, to make it easier to re-use your code, ...` or just leave it out. The `why` was already explained above.

**Comment by Keno** [comment]
Yes, there should not be an article in front of most of these nouns because they are abstract concepts. Adding the article would particularize them. E.g. consider the difference between `Code is art` (Coding is an art), `the code is art` (a particular piece of code has artistic value), `the code is the art` (in a particular piece of art we're discussing the artistic expression is in the code).

**Comment by logankilpatrick** [comment]
I don’t want to waste anyone’s time with this. I just saw the suggested changes when I was reading the style guide. I appreciate the input and such. The docs are readable imo so no urgent need to change these things.

**Pull request closed without merging**